hathor/metrics.py

```diff
@@ -187,6 +188,20 @@ def __init__(
         self._lc_collect_data = LoopingCall(self._collect_data)
         self._lc_collect_data.clock = reactor
 
+        # Dict that stores the sizes of each column-family in RocksDB, in bytes
+        self.rocksdb_cfs_sizes: Dict[bytes, float] = {}
```

Comment:
The type is already defined above along with the other ones. I think the comment with details about the units can be there, actually all other properties have descriptions inside the constructor but type annotation outside, we probably haven't been consistent with this.

Reply:
Or they can all be inside the constructor, I think it's fine. Ideally we could use dataclasses more to avoid these constructors, but I wouldn't bother with this here.

Reply:
I though it was a good idea indeed to turn the class into a `dataclass`, and since it seemed easy enough, I did it already: https://github.com/HathorNetwork/hathor-core/pull/485/commits/d5f0e12371d7cc6605e508b62ad6aa4194ed7562